### PR TITLE
support tpch

### PR DESCRIPTION
**Pull request by follitude**
PTAL @siddontang

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @huachaohuang

**Comment by siddontang** [comment]
remove all vebose

**Comment by siddontang** [comment]
Any intermediate statistics output?

**Comment by follitude** [comment]
only the execution time of each sql @siddontang

**Comment by siddontang** [comment]
only run once for one query?

**Comment by follitude** [comment]
you can specify the times by using -loops @siddontang

File: tpch/main.go [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+			log.Printf("[%d] finished query %d: %d rows returned after %4.2f seconds\n",
+				id, query, numRows, elapsed.Seconds())
+		}
+	}
+}
+
+func main() {
+	flag.Usage = usage
+	flag.Parse()
+	dbURL := fmt.Sprintf("tidb://%s:%s@tcp(%s:%d)/tpch", *user, *password, *host, *port)
```
Comment (siddontang) [inline tpch/main.go:137]: if we didn't create tpch database before, we may meet error, you can try it.

**Comment by siddontang** [comment]
do we need to import dbgen for the tpch @huachaohuang

**Comment by huachaohuang** [comment]
It depends on how we want to run it.
How long does it take to load all data?
If it can be loaded in a few minutes, I think it is OK to add it to benchbot as a [bench suite](https://github.com/pingcap/octopus/tree/master/benchbot/suite).

File: tpch/main.go (hunk 1 above)
Comment (siddontang) [reply tpch/main.go:137]: any update?

**Comment by siddontang** [comment]
@huachaohuang 

can we close it?

**Comment by huachaohuang** [comment]
I think so, we use the original TPC-H scripts now.

**Pull request closed without merging**